.circleci/config.yml

```diff
@@ -146,7 +146,7 @@ jobs:
           path: /tmp/test-results
       - store_artifacts:
           path: /testlogs
-          when: on_fail
+          when: always
```

Comment:
Does `on_fail` not always work? This creates a lot of artifacts for every build, including successful ones right?

Reply:
I chose to upload the raw output even on success so we can look through the logs if we want to. Artifacts are only retained for 30 days so I'm not concerned about costs.